content/en/news/releases/1.13.x/announcing-1.13/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Announcing Istio 1.12
+linktitle: 1.12
+subtitle: Major Update
+description: Istio 1.12 release announcement.
+publishdate: 2021-11-18
+release: 1.12.0
+skip_list: true
+aliases:
+    - /news/announcing-1.12
+    - /news/announcing-1.12.0
+---
+
+We are pleased to announce the release of Istio 1.13!
+
+{{< tip >}}
+Istio 1.13.0 is officially supported on Kubernetes versions `1.19` to `1.23`.
+{{< /tip >}}
+
+
+Here are some of the highlights of the release:
```

Comment:
other things I would add:
* ProxyConfig
* Continued improvements in Telemetry
* Lots of bug fixes and stability improvements (WG promotion, etc). This is partially a filler since we don't have that much, but we did have a ton of big bug fixes - more than normal

Overall this release is a lot smaller for various reasons so I wouldn't expect a huge list here

Reply:
The hostname support for ingress gateway is worthy mentioned as highlights as many have complained about the issue, for multicluster + VMs.

Also watch for intermediate cacerts change without reboot istiod.  Big deal for users who wants to use not self signed certs (which is everyone is production).

Reply:
The items in the comments have been added.